### PR TITLE
feat: pi installation script

### PR DESCRIPTION
**Pull request by reneaaron**

### CODE REVIEW

File: scripts/update.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+echo "🔃 Updating Alby Hub..."
+sudo systemctl stop albyhub
+
+cd /opt/albyhub
+rm -rf albyhub-backup
+mv app albyhub-backup
```
Comment (reneaaron) [inline scripts/update.sh:8]: Shouldn't we also backup the data directory here? Currently we just backup the app folder, right?

Comment (bumi) [reply scripts/update.sh:8]: true, we should do that.

File: scripts/update.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+wget https://nightly.link/getalby/nostr-wallet-connect-next/workflows/package-raspberry-pi/master/nostr-wallet-connect.zip
+
+unzip nostr-wallet-connect.zip -d app
```
Comment (bumi) [inline scripts/update.sh:11]: this seems now broken to me. why did you remove the second unzip. 
sadly it's double zipped.

Comment (reneaaron) [reply scripts/update.sh:11]: i removed the double zipping in the build
